.github/workflows/build.yml

```diff
@@ -15,5 +15,11 @@ jobs:
           dotnet-version: 5.0.*
       - name: Install dependencies
         run: dotnet restore -p:NuGetBuild=True
+      - name: Add package
+        id: add_package
+        run: |
+          curl https://raw.githubusercontent.com/Lombiq/GitHub-Actions/dev/scripts/add-github-sourcelink.sh > add-github-sourcelink.sh
```

Comment:
Oh no, there's really no other option? Can't reusable workflows use local scripts (what about maybe putting it into the .github folder too?)?

If it won't work, then plese copy it into a `run` step, what can be copied, but keep an sh file when we want to edit it (and work in a proper editor), with instructions on how to update this step (and instructions in the step on changing the file).

Reply:
> Oh no, there's really no other option? Can't reusable workflows use local scripts (what about maybe putting it into the .github folder too?)?

Nope, I've tested putting it into .github and .github/workflows too. You don't get the files in the reusable workflow's directory, so Github itself is probably just directly accessing the workflow file.

> If it won't work, then plese copy it into a run step

Are you sure? I think that's significantly worse in terms of maintainability. What is your concern that drives this extreme move?

Reply:
Apart from the awkwardness of loading part of the pipeline via HTTP separately this also prevents using anything else from the consumer code apart from the latest `dev` version. While this might be what we'll do by default and usually, it's not necessarily the case every time, nor for other users of this projects. Specifying anything else apart from `@dev` [when calling the workflow](https://github.com/Lombiq/Helpful-Libraries/blob/issue/INFRA-92/.github/workflows/publish.yml#L10) would result in confusion, the YML file being the correct version and the script something else.

Reply:
Ok I did that. It nooks genuinely disturbing. Sounds like we are making regular operation much worse just to serve some niche hypothetical.

Reply:
For the regular operation this will result in one fewer HTTP call which I'd argue is better :). Editing the script, yes.

Reply:
I would argue that an HTTP request is cheap in comparison to copying file content by hand on every change, and more so, remembering it!

I don't see the advantage.

Reply:
I was referring to the regular operation. While I wouldn't care too much about an HTTP request (though it's one additional moving part that can have hiccups any time) this is not my point, see above. This file should change rarely, and when it does, what I've written above will be important.